docs/v1.md

```diff
@@ -175,25 +222,34 @@
 - hasDescription() : `boolean`
 - hasSummary() : `boolean`
 - id() : `string`
-- isApplicationPublishing() : `boolean`
-- isApplicationSubscribing() : `boolean`
-- isClientPublishing() : `boolean`
-- isClientSubscribing() : `boolean`
```

Comment:
In 2.x.x these methods would then have a different meaning, so for now I prefer not to add these methods and see how 3.x.x will look like and then we will add the appropriate methods.

Reply:
But here we are building an API based on intents, and those are still valid. I think we should add them and, for 2.x.x, return only the value it should be returned, without the `application` or `client` perspective.

Reply:
Do you suggest `isSubscribing`, `isPublishing`?

Reply:
No, I suggest we do, at implementation level:


### 2.x.x:

- `isSend()` -> this.action === `publish`
- `isReceive()` ->  this.action === `subscribe`  

### 3.x.x:

- `isSend()` -> this.action === `send`
- `isReceive()` ->  this.action === `receive`

Reply:
@smoya Thanks, however in in the `2.x.x` we should switch publish/subscribe meaning, because `isSend` and `isReceive` describes operation from application point of view, not user, so `publish` -> `receive` (user publishes something, so app receives it), but `subscribe` -> `send` (user subscribed to something, so app sends it to the user), but I could be wrong, so please correct me.

Reply:
Yes, I thought about this but I thought we could just follow the perspective or point of view each spec version has:

- 2.x.x: described from the point of view of the **client**
- 3.x.x: described from the point of view of the **application**

However, if we start documenting each method on the Parser-API, it could be convenient to clarify what is the point of view of the API itself, which aligned with >=3.x.x, It would be the **application**'s one.

I think I can agree we should go further and do what you suggest @magicmatatjahu, to be completely strict on meanings and explicitly consider `send` **always** means the application is sending a message, no room for interpretation 👍 .

WDYT @fmvilas @jonaslagoni ?